packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -932,7 +932,7 @@ $bpk-spacing-v2: true;
   display: inline-block;
   width: 100%;
   max-width: 100%;
-  min-height: $bpk-textarea-min-height;
```

Comment:
I guess there will be a foundations PR to follow with this token being deprecated?

Reply:
Yes! Alternatively, we can keep the token and change its value as it is being used in one other project